app/web/package.json

```diff
@@ -22,41 +22,41 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@date-io/dayjs": "1.x",
-    "@emotion/cache": "^11.13.1",
     "@emotion/react": "^11.13.5",
-    "@emotion/server": "^11.11.0",
     "@emotion/styled": "^11.13.5",
     "@fontsource/mansalva": "^5.1.0",
     "@fontsource/ubuntu": "^5.1.0",
     "@mui/icons-material": "^5.16.8",
     "@mui/lab": "^5.0.0-alpha.164",
     "@mui/material": "^5.16.7",
     "@mui/styles": "^5.16.7",
+    "@mui/system": "^5.4.1",
     "@mui/utils": "^5.16.6",
     "@mui/x-date-pickers": "^5.0.20",
     "@sentry/nextjs": "^6.19.7",
     "@sentry/react": "^8.41.0",
     "@stripe/stripe-js": "^5.2.0",
     "@toast-ui/editor": "^3.2.2",
-    "@types/react-simple-maps": "^3.0.6",
     "classnames": "^2.5.1",
     "dayjs": "^1.11.13",
     "frontmatter-markdown-loader": "^3.7.0",
+    "geojson": "^0.5.0",
     "glob": "^8.0.3",
     "google-protobuf": "^3.21.4",
     "grpc-web": "1.5.0",
+    "i18next": "^21.5.3",
```

Comment:
We're importing it in app/web/.storybook/i18n.ts but never installed the package.